<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>GpgGenBase</class>
  <widget class="QDialog" name="GpgGen" >
    <property name="geometry" stdset="0" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>385</width>
        <height>241</height>
      </rect>
    </property>
    <property name="windowTitle" stdset="0" >
      <string>Generate key</string>
    </property>
    <property name="sizeGripEnabled" stdset="0" >
      <bool>true</bool>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" stdset="0" >
        <number>11</number>
      </property>
      <property name="spacing" stdset="0" >
        <number>6</number>
      </property>
      <item row="1" column="0" >
        <widget class="QLabel" name="TextLabel3" >
          <property name="text" stdset="0" >
            <string>EMail:</string>
          </property>
          <property name="alignment" stdset="0" >
            <set>Qt::AlignVCenter|Qt::AlignRight</set>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="2" column="0" >
        <widget class="QLabel" name="TextLabel4" >
          <property name="text" stdset="0" >
            <string>Comment:</string>
          </property>
          <property name="alignment" stdset="0" >
            <set>Qt::AlignVCenter|Qt::AlignRight</set>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="0" column="0" >
        <widget class="QLabel" name="TextLabel2" >
          <property name="text" stdset="0" >
            <string>Name:</string>
          </property>
          <property name="alignment" stdset="0" >
            <set>Qt::AlignVCenter|Qt::AlignRight</set>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="0" column="1" >
        <widget class="QLineEdit" name="edtName" />
      </item>
      <item row="2" column="1" >
        <widget class="QLineEdit" name="edtComment" />
      </item>
      <item row="1" column="1" >
        <widget class="QComboBox" name="cmbMail" />
      </item>
      <item row="3" column="0" >
        <widget class="QLabel" name="TextLabel1" >
          <property name="text" stdset="0" >
            <string>Passphrase:</string>
          </property>
          <property name="alignment" stdset="0" >
            <set>Qt::AlignVCenter|Qt::AlignRight</set>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="4" column="0" >
        <widget class="QLabel" name="TextLabel2_2" >
          <property name="text" stdset="0" >
            <string>Retype passphrase:</string>
          </property>
          <property name="alignment" stdset="0" >
            <set>Qt::AlignVCenter|Qt::AlignRight</set>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="3" column="1" >
        <widget class="QLineEdit" name="edtPass1" >
          <property name="echoMode" stdset="0" >
            <enum>QLineEdit::Password</enum>
          </property>
        </widget>
      </item>
      <item row="4" column="1" >
        <widget class="QLineEdit" name="edtPass2" >
          <property name="echoMode" stdset="0" >
            <enum>QLineEdit::Password</enum>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="5" column="0" colspan="2" >
        <widget class="QLabel" name="lblProcess" >
          <property name="sizePolicy" stdset="0" >
            <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>7</vsizetype>
            </sizepolicy>
          </property>
          <property name="text" stdset="0" >
            <string/>
          </property>
          <property name="alignment" stdset="0" >
            <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="6" column="0" colspan="2" >
        <layout class="QHBoxLayout" >
          <property name="margin" stdset="0" >
            <number>0</number>
          </property>
          <property name="spacing" stdset="0" >
            <number>6</number>
          </property>
          <item>
            <spacer name="Horizontal Spacing2" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="buttonOk" >
              <property name="text" stdset="0" >
                <string>&amp;OK</string>
              </property>
              <property name="autoDefault" stdset="0" >
                <bool>true</bool>
              </property>
              <property name="default" stdset="0" >
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="buttonCancel" >
              <property name="text" stdset="0" >
                <string>&amp;Cancel</string>
              </property>
              <property name="autoDefault" stdset="0" >
                <bool>true</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <tabstops>
    <tabstop>edtName</tabstop>
    <tabstop>cmbMail</tabstop>
    <tabstop>edtComment</tabstop>
    <tabstop>edtPass1</tabstop>
    <tabstop>edtPass2</tabstop>
    <tabstop>buttonOk</tabstop>
    <tabstop>buttonCancel</tabstop>
  </tabstops>
  <connections>
    <connection>
      <sender>buttonOk</sender>
      <signal>clicked()</signal>
      <receiver>GpgGen</receiver>
      <slot>accept()</slot>
    </connection>
    <connection>
      <sender>buttonCancel</sender>
      <signal>clicked()</signal>
      <receiver>GpgGen</receiver>
      <slot>reject()</slot>
    </connection>
  </connections>
</ui>
